# HG changeset patch
# User Tatsuro MATSUOKA <tmacchant@yahoo.co.jp>
# Date 1295763097 28800
#      Sat Jan 22 22:11:37 2011 -0800
# Branch release-3-4-x
# Node ID dfe958c044a26e703baf2f20b69249f76bbac5fd
# Parent  097c84d2f87b90753473858391309172a89c8bd6
Small corrections to README.MinGW.

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-22  Tatsuro MATSUOKA  <tmacchant@yahoo.co.jp>
+
+	* README.MinGW: Small corrections to documentation.
+
 2010-01-22  Rik  <octave@nomad.inbox5.com>
 
 	* README.devel: Update for 3.4 release.
 
 2010-01-22  Rik  <octave@nomad.inbox5.com>
 
 	* HACKING: Revise and incorporate all of file ROADMAP.
 	* ROADMAP: Deleted.
diff --git a/README.MinGW b/README.MinGW
--- a/README.MinGW
+++ b/README.MinGW
@@ -42,17 +42,17 @@ Additional information beyond the tutori
  (Compiling for 64-bit binaries requires mingw-64 toolkits.  This has not been
   tested.)
 
  2.2 Dependencies
  ----------------------
 
  External program dependencies
  
-  * gnuplot (you can build this with the shell script in Howto.txt)
+  * gnuplot (you can build this with the shell script mentioned in Howto.txt)
   * MikTeX
   * ghostscript
   * pstoedit (required for printing using FLTK (OpenGL) graphics)
   * fig2dev (Optional; better quality for FLTK printing in some file formats)
   * sed
   * less
 
  The tools below are required for building the development sources from
@@ -61,17 +61,17 @@ Additional information beyond the tutori
 
   * Mercurial
   * git
   * automake
   * autoconf
 
  Library dependencies
 
-  Please read Howto.txt carefully on how to build them.  The Howto.txt is ols
+  Please read Howto.txt carefully on how to build them.  The Howto.txt is old
   so some revisions will be required when using GCC-4.5.0
 
  2.3 Download Octave source
  ----------------------
 
   The source for testing snapshots can be downloaded from the official Octave
   web site.  http://www.gnu.org/software/octave/download.html 
 
